src/dsp_tools/utils/project_create.py

```diff
@@ -46,7 +46,7 @@ def _create_project_on_server(
     try:
         # the normal, expected case is that this try block fails
         project_local = Project(con=con, shortcode=project_definition["project"]["shortcode"])
-        project_remote: Project = try_network_action(lambda: project_local.read())
```

Comment:
it was necessary to refactor `try_network_action()` because of https://pylint.pycqa.org/en/latest/user_guide/messages/warning/cell-var-from-loop.html.
In the case of `try_network_action()`, I don't think there was ever a danger, but I think it's important for me to be warned of this potentially dangerous error. 
So I found it necessary to refactor all occurrences where pylint warns me of this potential error, so that if I receive such a warning in the future, I know that I have to take it seriously.